commands/conduit_cli.py

```diff
@@ -11,36 +12,63 @@
 
 
 def get_cluster_arn(app: str, env: str) -> str:
-    client = boto3.client("resourcegroupstaggingapi")
+    ecs_client = boto3.client("ecs")
+    clusters_response = ecs_client.list_clusters()
```

Comment:
The `response` bit sounds redundant... I feel like this could be `ecs_clusters`.

```suggestion
    ecs_clusters_ = ecs_client.list_clusters()
```

Reply:
Could you go straight for this here?
```suggestion
    cluster_arns = ecs_client.list_clusters()["clusterArns"]
```